### PR TITLE
Add Target User Profile

### LINKED ISSUE #7: Update the DG: Target user profile

### PR DESCRIPTION
**Pull request by s7u4rt99**
Resolves #7

### CODE REVIEW

**Comment by yourally2** [comment]
LGTM!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -257,11 +257,26 @@ _{Explain here how the data archiving feature will be implemented}_
...
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+* Name: Kevin Bernard Long Zheng Wei
+* Age: 28
+* Gender: Male
+* Marital status: Single
+* Country: Singapore
+* Habits: Sleeping, Netflix, Gym, Gaming
+* Lifestyle: Code, eat, sleep, give part time tuition
```
Comment (softmagnet) [inline docs/DeveloperGuide.md:266]: Shouldn't the target user be a full-time tutor?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -257,11 +257,26 @@ _{Explain here how the data archiving feature will be implemented}_
...
+    * prefer desktop apps over other types
+    * can type fast
+    * prefers typing to mouse interactions
+    * is reasonably comfortable using CLI apps
+* Interest: Coding, building apps during his free time with friends and teaching people
+* Values: Work hard play hard
+* Salary: $12000/mth
+* Job: Full time A levels Mathematics tuition teacher (group and one on one) at his own new tuition company
```
Comment (softmagnet) [inline docs/DeveloperGuide.md:274]: he manages a company and teaches so many tuitions at the same time? Is this a bit unrealistic?

**Review by softmagnet — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (s7u4rt99) [reply docs/DeveloperGuide.md:274]: Ok will change them

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (s7u4rt99) [reply docs/DeveloperGuide.md:266]: Yes it should, forgot to change it

**Comment by Ongzl** [comment]
LGTM

**Pull request merged**